Bottom: 08ad22776c846f8c45c113865ec8f3a447e9f24f
Top:    2573f1acb8a9cc340c5d34f6975601668077a654
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-13 15:09:55 +0100

Temp build fixes (NOT NECESSARILY RIGHT!)


---

diff --git a/src/hid/gtk/gui-drc-window.c b/src/hid/gtk/gui-drc-window.c
index fa2ec86..c9916d4 100644
--- a/src/hid/gtk/gui-drc-window.c
+++ b/src/hid/gtk/gui-drc-window.c
@@ -166,13 +166,13 @@ unset_found_flags (int AndDraw)
       }
   }
   ENDALL_LOOP;
-  COPPERPOLYGON_LOOP (PCB->Data);
+  COPPERPOUR_LOOP (PCB->Data);
   {
-    if (TEST_FLAG (flag, polygon))
+    if (TEST_FLAG (flag, pour))
       {
-	AddObjectToFlagUndoList (POLYGON_TYPE, layer, polygon, polygon);
-	CLEAR_FLAG (flag, polygon);
-	DrawPolygon (layer, polygon, 0);
+	AddObjectToFlagUndoList (POUR_TYPE, layer, pour, pour);
+	CLEAR_FLAG (flag, pour);
+	DrawPour (layer, pour, 0);
 	change = True;
       }
   }
